arch/risc-v/src/esp32c3/hardware/esp32c3_rtccntl.h

```diff
@@ -49,13 +49,13 @@
  * write-enable the wdt registers
  */
 
-#define RTC_CNTL_WDT_WKEY_VALUE 0x50D83AA1
+#define RTC_CNTL_WDT_WKEY_VALUE     0x50D83AA1
```

Comment:
```suggestion
#define RTC_CNTL_WDT_WKEY_VALUE     0x50d83aa1
```
It is common practice to use lowercase letters for hex literals.